.github/CONTRIBUTING.md

```diff
@@ -28,6 +28,24 @@ IGListExperimentEnabled(self.experiments, IGListExperimentMyAwesomeChange) {
 
 Once your experiment is confirmed we will remove the option and wrapping check!
 
+## How we do major and minor releases
+
+Current release pull requests merge into stable.
+Next major release pull requests merge into master.
+
+Then we pull request from stable to master so we can keep the git history clean.
+
+Example:
+
+if current release is 2.1.0, then any commits for 2.2.0 go into stable while commits for 3.0 would go to master.
```

Comment:
Ack we should have updated the issue. This has changed. The workflow now is:

- Everything merges into `master`
- When we cut a release, we merge from `master` into `stable`, tag, and push to CocoaPods. 

Sorry for the churn!

Also can we wrap branch names in markdown code formatting?